src/components/NamePicker.tsx

```diff
@@ -9,10 +9,10 @@ import { makeStyles } from '@material-ui/core/styles';
 
 export default function NamePicker(): JSX.Element {
   const classes = useStyles();
-  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
-  const [anchorEl2, setAnchorEl2] = useState<null | HTMLElement>(null);
-  const [disabled1, setDisabled1] = useState(true);
-  const [disabled2, setDisabled2] = useState(true);
+  const [anchorEl, setAnchorEl] = useState<undefined | HTMLElement>(undefined);
+  const [anchorEl2, setAnchorEl2] = useState<undefined | HTMLElement>(undefined);
```

Comment:
At the risk of making these variables even longer... since you are using 'first' and 'last' in the other names, you probably should do that here, too. Maybe just 'anchorFirst' or 'firstAnchor'?

Reply:
Updated to firstAnchor, setFirstAnchor, lastAnchor, and setLastAnchort. Thanks! :)